lib/spack/spack/build_systems/windows_variants.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.directives import variant
+from spack.package import Package
+
+
+class WindowsPackage(Package):
```

Comment:
Is this class actually handling *windows* options or *msvc* options? I could be misunderstanding this but there seems to be an assumption that windows = msvc which I don't believe is an assumptions spack should be making.